server/src/main/java/org/solarbank/server/CreateMockCalculateRequest.java

```diff
@@ -0,0 +1,32 @@
+package org.solarbank.server;
+
+import org.solarbank.server.dto.CalculateRequest;
+import org.solarbank.server.dto.EnergyTariff;
+import org.solarbank.server.dto.Location;
+import org.solarbank.server.dto.PanelSize;
+
+public class CreateMockCalculateRequest {
```

Comment:
added this class to construct a mock CalculateRequest object which is used in tests